src/map/lua/luautils.cpp

```diff
@@ -5316,20 +5316,22 @@ namespace luautils
         }
     }
 
-    bool OnChocoboDig(CCharEntity* PChar, bool pre)
+    bool OnChocoboDig(CCharEntity* PChar)
     {
         TracyZoneScoped;
 
-        auto name = PChar->loc.zone->getName();
-
+        // Check if zone can be digged in.
+        auto name         = PChar->loc.zone->getName();
         auto onChocoboDig = lua["xi"]["zones"][name]["Zone"]["onChocoboDig"];
```

Comment:
Instead of modifying every zone where digging could happen, would it be better/easier to just have a table in the chocobo digging global that lists all the zones where it's possible? Then in `onChocoboDig` you can check to see if the player is in that zone - then you're done, no extra changes required apart from in the global - also easier to modify if people want that

Reply:
I would say get rid of the onChocoboDig() functions in all the zone files, we're gearing up to do this sort of transformation with zone mixins too